src/config.js

```diff
@@ -33,8 +33,16 @@ module.exports = {
     hostname: process.env.ANALYTICS_HOSTNAME || "",
   },
 
+  postgres: {
+    host : process.env.POSTGRES_HOST,
+    user : process.env.POSTGRES_USER,
+    password : process.env.POSTGRES_PASSWORD,
+    database : process.env.POSTGRES_DATABASE || "analytics-reporter",
+  },
+
   static: {
-    path: '../analytics.usa.gov/'
+    path: '../analytics.usa.gov/',
   },
 
+  timezone: process.env.ANALYTICS_TIMEZONE || "US/Eastern",
```

Comment:
This makes sense, and I understand why it's included here, but it is possible that this change will cause regressions for other users of the tool which aren't located in the Eastern time zone.

Could we keep all the timezone additions, but make it default to whatever it was likely defaulting to before (the local system's time zone, I think)? That way we could override it as `US/Eastern` for our use case, but hopefully minimize disturbance to others.